docs/_docs/user-guide/eldritch.md

```diff
@@ -441,8 +446,8 @@ An example is below:
 The <b>sys.get_pid</b> method returns the process ID of the current process.
 An example is below:
 
-```json
-$> sys.get_pid()
+```python
+> sys.get_pid()
```

Comment:
This line doesn't have the `$` like the other examples

Reply:
Ooh good catch.
I'm not sure how either will render but we'll find out 🤞 if it looks bad i'll try something different with these two.